pkg/test/utils/kubernetes.go

```diff
@@ -216,12 +214,14 @@ func LoadYAML(path string) ([]runtime.Object, error) {
 }
 
 // InstallManifests recurses over ManifestsDir to install all resources defined in YAML manifests.
-func InstallManifests(ctx context.Context, client client.Client, dClient discovery.DiscoveryInterface, manifestsDir string) error {
+func InstallManifests(ctx context.Context, client client.Client, dClient discovery.DiscoveryInterface, manifestsDir string) ([]runtime.Object, error) {
+	objects := []runtime.Object{}
+
 	if manifestsDir == "" {
-		return nil
+		return objects, nil
```

Comment:
shouldn't this be considered an error? 🤔

Reply:
No, if the user doesn’t provide manifests, then we have no work to do.